src/Cohesity.Powershell/Scripts/Recover-CohesityBackupToView.ps1

```diff
@@ -0,0 +1,117 @@
+function Recover-CohesityBackupToView {
+    [CmdletBinding()]
+    Param(
+        [Parameter(Mandatory = $false)]
+        [String]$SourceName,
+        [Parameter(Mandatory = $false)]
+        [String]$ViewName,
+        [Parameter(Mandatory = $false)]
+        [String]$QOSPolicy,
```

Comment:
Use validate set right here. You can provide a set of values outside which the value set for the variable would fail. So you wouldn't have to validate it .. and user would know upfront